src/App.js

```diff
@@ -1,10 +1,31 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
+import Button from './Components/Button';
 import './App.css';
 
 function App() {
+  const [activeBtn, setActiveBtn] = useState("")
+
+  const handleClick = (button) => {
+    setActiveBtn(button)
+  };
+                             
   return (
-    <div className="App">
-      
+    <div className='todo-container'>
+      <h1>TODO APP</h1>   
+      <Button 
+        label="Add Task" 
+        onClick={() => handleClick("add")}
+        backgroundColor="#1a5e66"
+        isActive= {activeBtn === "add"}
+        isDisabled={false}
+      />
+      <Button 
+        label="View Task" 
+        onClick={() => handleClick("view")}
+        backgroundColor="#1a5e66"
+        isActive={activeBtn === "view"}
+        isDisabled={false}
```

Comment:
No need to pass if the value of isDisabled is false. we have already set the default value as `false` right?